File: app/src/main/java/com/example/ruslanyussupov/popularmovies/data/local/FavouriteMoviesDb.java
package com.example.ruslanyussupov.popularmovies.data.local;

import android.arch.persistence.room.Database;
import android.arch.persistence.room.RoomDatabase;

import com.example.ruslanyussupov.popularmovies.data.model.Movie;

@Database(entities = {Movie.class}, version = 1, exportSchema = false)
public abstract class FavouriteMoviesDb extends RoomDatabase {

    public abstract MovieDao movieDao();

}

File: app/src/main/java/com/example/ruslanyussupov/popularmovies/data/local/FavouriteMoviesDb.kt
package com.example.ruslanyussupov.popularmovies.data.local

import androidx.room.Database
import androidx.room.RoomDatabase

import com.example.ruslanyussupov.popularmovies.data.model.Movie

@Database(entities = [Movie::class], version = 1, exportSchema = false)
abstract class FavouriteMoviesDb : RoomDatabase() {

    abstract fun movieDao(): MovieDao

}
